akka-docs/src/main/paradox/stream/operators/Sink/collect.md

```diff
@@ -0,0 +1,26 @@
+# Sink.collect
+
+Collect all input elements using a Java @javadoc[Collector](java.util.stream.Collector).
+
+@ref[Sink operators](../index.md#sink-operators)
+
+## Signature
+
+@apidoc[Sink.collect](Sink$) { java="#collect(java.util.stream.Collector)" }
```

Comment:
This will link to https://doc.akka.io/api/akka/2.6/akka/stream/scaladsl/Sink$.html in Scala mode, where `collect` doesn't exist. Perhaps in Scala mode we should point to `Sink.seq` or so? I'm kinda OK with this as well though.

Reply:
But a Java Collector can collect to a Map too, should we add an overloaded method to scaladsl too?